README.md

```diff
@@ -109,7 +109,7 @@ build.
 
    # Install build tools needed for Rust. If you're building a 32-bit compiler,
    # then replace "x86_64" below with "i686". If you've already got git, python,
-   # or CMake installed and in PATH you can remove them from this list. Note
+   # or CMake installed, and in PATH, you can remove them from this list. Note
```

Comment:
This is less clear IMO - it makes it look like these clauses are part of the list, not separate ideas.